CHANGELOG.md

```diff
@@ -1,8 +1,31 @@
+## 0.5.0
+
+### New Additions
+
+* Much improved test helpers ([#109])
+* `Bamboo.TaskSupervisorStrategy` is now started by default ([#133])
+* Add Mailgun adapter ([#125])
+* Link to new Sparkpost adapter ([#118])
+* Shared mode for working with multiple process/acceptance tests ([#136])
+* New `put_layout/2` for setting HTML and text layouts at the same time ([#122])
+
+### Fixes
+
+* Show correct from address in `EmailPreview` ([#127])
+
+[#109]: https://github.com/thoughtbot/bamboo/pull/109/files
+[#133]: https://github.com/thoughtbot/bamboo/pull/133/files
+[#125]: https://github.com/thoughtbot/bamboo/pull/125/files
+[#118]: https://github.com/thoughtbot/bamboo/pull/118/files
+[#136]: https://github.com/thoughtbot/bamboo/pull/136/files
+[#122]: https://github.com/thoughtbot/bamboo/pull/122/files
+[#127]: https://github.com/thoughtbot/bamboo/pull/127/files
```

Comment:
Yes there were a couple. I added them in the latest push
> On Apr 29, 2016, at 4:12 PM, Tute Costa <notifications@github.com> wrote:
> 
> In CHANGELOG.md <https://github.com/thoughtbot/bamboo/pull/140#discussion_r61637996>:
> 
> > +* Add Mailgun adapter ([#125])
> > +* Link to new Sparkpost adapter ([#118])
> > +* Shared mode for working with multiple process/acceptance tests ([#136])
> > +* New `put_layout/2` for setting HTML and text layouts at the same time ([#122])
> > +
> > +### Fixes
> > +
> > +* Show correct from address in `EmailPreview` ([#127])
> > +
> > +[#109]: https://github.com/thoughtbot/bamboo/pull/109/files
> > +[#133]: https://github.com/thoughtbot/bamboo/pull/133/files
> > +[#125]: https://github.com/thoughtbot/bamboo/pull/125/files
> > +[#118]: https://github.com/thoughtbot/bamboo/pull/118/files
> > +[#136]: https://github.com/thoughtbot/bamboo/pull/136/files
> > +[#122]: https://github.com/thoughtbot/bamboo/pull/122/files
> > +[#127]: https://github.com/thoughtbot/bamboo/pull/127/files
> Any backwards incompatible changes to mention?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/thoughtbot/bamboo/pull/140/files/aeb397335ce0f38bad3ca76c2fdc7551d812cf69#r61637996>